applivery-base/build.gradle

```diff
@@ -26,7 +26,7 @@ android {
     buildToolsVersion "29.0.2"
 
     defaultConfig {
-        minSdkVersion 21
+        minSdkVersion 15
```

Comment:
In the issue description you proposed to change the min sdk version to android 4.4 (api 19) but here is api 15. What is the minimum api what you want?

Reply:
Hi, thanks for taking this into consideration.

We currently support as low as android 4.0.3 (api 15) and for us, it would be great if you could do the same.
However, if you can't go that low, for us it would be enough if you lower it to android 4.4 (api 19) since most of our customers are using still using that version.